index.html

```diff
@@ -2,18 +2,29 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <title>Document</title>
+    <title>quiz app</title>
     <link rel="stylesheet" href="styles.css" />
   </head>
+
   <body>
     <header>
-      <h2>Quiz App</h2>
+      <h1>Quiz App</h1>
     </header>
-    <section>
-      <h3>Very hard question</h3>
-      <h4>show me the answer</h4>
-      <h6 class="tag1">bookmark1</h6>
-      <h6 class="tag2">bookmark2</h6>
-    </section>
+
+    <main>
+      <section class="q1">
```

Comment:
```suggestion
      <section class="question-card">
```
I would also a more descriptive class name, also we avoid shortcuts. And why you gave your question cards different classes ("q1" and "q2"), do you plan different styling for each question card? If not, the could have the same class.